### PR TITLE
Chainbridge Generic ERC20

### PR DESCRIPTION
**Pull request by frankli-dev**

### CODE REVIEW

**Review by Gauthamastro — COMMENTED** [review]
It's good

File: runtime/Cargo.toml [hunk 1]
```diff
@@ -95,7 +114,10 @@ std = [
 	"polkadex-fungible-assets/std",
 	"polkadex-ocex/std",
 	"pallet-substratee-registry/std",
-	"pallet-proxy/std"
+	"pallet-proxy/std",
+	"chainbridge/std",
+	'erc721/std',
+    'example/std',
```
Comment (ayushmishra2005) [inline runtime/Cargo.toml:120]: formatting

Comment (Gauthamastro) [reply runtime/Cargo.toml:120]: @frankli-dev @mambisi @ksr30 Please use rust-fmt before upload. It solves these issues

Thanks @ayushmishra2005

Comment (frankli-dev) [reply runtime/Cargo.toml]: done. (I've installed TOML formatter to my vscode)

**Review by Gauthamastro — APPROVED** [review]

**Pull request merged**